File: javascript/reversed_strings.js
// Complete the solution so that it reverses the string passed into it.

// example: 'world'  =>  'dlrow' 'word'   =>  'drow'

let str = 'word'

function reversed(str){
	return str.split("").reverse().join("");
}

s2 = reversed(str)
console.log(s2)

File: python/reversed_strings.py
# Complete the solution so that it reverses the string passed into it.

# 'world'  =>  'dlrow'
# 'word'   =>  'drow'

str = 'stressed'
# original solution
# reversed = str[::-1]

# alternate solution
reversed = list(str)
reversed.reverse()
reversed = ''.join(reversed)
print(reversed)


